### PR TITLE
switched to development version of multiqc to work around a bug in th…

### PR DESCRIPTION
**Pull request by dansomers**
…e stable release, added HTSeq per Wei's request, threw in a pip upgrade before pip installs

### CODE REVIEW

File: rnaseq/tasks/main.yml [hunk 1]
```diff
@@ -89,15 +89,23 @@
...
+    extra_args: --upgrade
 - name: Install cutadapt
   become: yes
   pip:
     name: cutadapt
     version: 1.14
-- name: Install MultiQC
+- name: Install development version of MultiQC
+  become: yes
+  shell: 'pip install --upgrade --force-reinstall git+https://github.com/ewels/MultiQC.git'
```
Comment (hlapp) [inline rnaseq/tasks/main.yml]: So we can't use the `pip` module for that?

**Comment by dansomers** [comment]
I tried that approach first, but the pip module couldn't work with the package name (pip threw an error when run via the ansible module but worked in shell).

-Dan

Dan Somers
Linux Systems Administrator
Center for Genomic and Computational Biology
(919) 668-3565
daniel.somers@duke.edu

________________________________________

**Comment by hlapp** [comment]
> I tried that approach first, but the pip module couldn't work with the package name (pip threw an error when run via the ansible module but worked in shell).

Are you sure you gave that in the correct way? The documentation for the pip module contains among its example exactly the case that we need, I think:
http://docs.ansible.com/ansible/latest/pip_module.html

```
# Install MyApp using one of the remote protocols (bzr+,hg+,git+).
- pip:
    name: git+http://myrepo/app/MyApp
```

**Comment by dansomers** [comment]
That's what I tried (Dan L provided an example from one of his roles) but that task errors out with:

fatal: [localhost]: FAILED! => {"changed": false, "cmd": "/bin/pip2 install --upgrade --force-reinstall -e git+https://github.com/ewels/MultiQC.git", "failed": true, "msg": "\n:stderr: Could not detect requirement name, please specify one with #egg=\n"}

Appending #egg=multiqc to the name fixes this.

Dan Somers
Linux Systems Administrator
Center for Genomic and Computational Biology
(919) 668-3565
daniel.somers@duke.edu

________________________________________

**Comment by dleehr** [comment]
👍

**Pull request merged**